tests/unit/commands/index-command-test.php

```diff
@@ -171,6 +181,8 @@ public function test_execute() {
 				->andReturn( \array_fill( 0, 25, true ), \array_fill( 0, 5, true ) );
 		}
 
+		$this->indexable_helper->expects( 'should_index_indexables' )->once()->andReturn( true );
```

Comment:
There is no case checking where this is false, I think that would be nice to also test. just to make sure the other code is not executed.

Reply:
Done 👍